lib/active_fedora/attribute_methods.rb

```diff
@@ -13,7 +13,7 @@ def self.set_name_cache(name, value)
       end
     end
 
-    BLACKLISTED_CLASS_METHODS = %w(private public protected allocate new name parent superclass).freeze
+    DANGEROUS_CLASS_METHODS = %w(private public protected allocate new name parent superclass).freeze
```

Comment:
As this was copied from ActiveRecord, maybe follow the constant they have chosen: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/attribute_methods.rb#L26